src/allocation.rs

```diff
@@ -131,7 +151,7 @@ pub fn try_zeroed_slice_box<T: Zeroable>(
     let slice =
       unsafe { core::slice::from_raw_parts_mut(ptr as *mut T, length) };
```

Comment:
This should be `core::ptr::slice_from_raw_parts_mut()` to avoid creating an intermediate reference (also also reduces the amount of unsafe by one line).